DEVELOPERS.md

```diff
@@ -54,19 +54,13 @@ To contribute code to Ocular, you must fork the [Ocular repo](https://github.com
       turbo build
    ```
 
-2. Copy the example `env.dev.example` to `.env.dev`
-
-   ```sh
-      cp env.dev.example packages/ocular/.env.dev
-   ```
-
-3. Navigate to Ocular Dir
+2. Navigate to Ocular Dir
```

Comment:
```sh
      cp env.dev.example packages/ocular/.env.dev
   ```
   
   this command copy the required file to ocular directory

Reply:
env.dev.example does not exist